stake-pool/cli/src/main.rs

```diff
@@ -98,9 +101,12 @@ fn send_transaction(
     Ok(())
 }
 
+#[allow(clippy::too_many_arguments)]
 fn command_create_pool(
```

Comment:
For pool creation, I feel that a better way could be to provide a prototypical stake account.  The program then examines that stake account on-chain and copies over the Lockup.  This way there's 0% chance of user error when specifying the lockup parameters.